### DISCUSSION TITLE
Problem Trigger-Node - Entity-ID from flow-variable

### DISCUSSION THREAD

**Discussion by S-E-B-123** [zachowj/node-red-contrib-home-assistant-websocket#692] [Q&A]
Hi,
i have a flow variable `var1` with value `light.test_light`. If i use `{{flow.var1}}` as Entity-ID the `Trigger: state`-Node does not trigger if i switch my test_light. If i use the entity name directly it is working.  Any idea where my problem is?

**Comment by zachowj** [comment]
The entity id field of the trigger state node does not accept mustache templates.

**Reply by S-E-B-123** [reply]
Ohh, than the Node documentation needs to be updated.
[[](url)](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/trigger-state.html#trigger-state)

```
Entity ID required  string

Accepts [Mustache Templates]

The id of an entity to use for the comparison.
Custom ids can be inserted into the list by adding a # at the end of the id
```
